README.md

```diff
@@ -1,2 +1,350 @@
+
+![listing the /proc directory content](/images/proc_dir.png)
+
+In this sense, we did not specify that but you may notice that **`/dev`** has the same feel to it as **`/proc`** too. And yes, it is a virtual directory too. **`/proc`** is a vital directory for us because we will get in here and establish our own cgroups and more. We will come to them later with utmost detail.
+
+---
+
+Let's check if this 'super user' guy is really super or a lamer with no life... 🧐
+
+![ls root](/images/lsa_root.png)
+
+**`/root`** is equivalent to `root` - a.k.a. administrator or super - users **`/home`**. 
+
+---
+
+![run lsa](/images/run_ls.png)
+
+**`/run`** is the run-time variable data. It is a temporary location for system processes to store their own temporary data. When you boot your system, this folder is probably empty and starts to get populated as your system operates.  It is relatively a new directory. Like **`/proc`**, it stores run-time volatile data.
+
+---
+
+This one is pretty huge too...
+
+**`/sbin`** has the same duty as the **`/bin`** directory but it is for the superuser - 's' in the beginning for that reason -. You use those programs with the `sudo` command which **temporarily** renders you "super", and gives you powers of the superuser.
+As you may infer, **`/sbin`** contains mutators, and programs that change something within the system such as installing, deleting, or formatting/changing a component or package. The essential binaries to boot, restore, recover, and repair the system are here too. For example, updating passwords for users `chgpasswd [group_name:password]` resides here. 
+
+![chgpasswd place](/images/chgpasswd_place.png)
+
+---
+
+![lsa snap ](/images/lsa_snap.png)
+
+**`/snap`** is for storing your favorite Snapchat stories. I am kidding. Let us read what the `README` says within the **`/snap`** directory. 
+
+![snap readme](/images/snap_readme.png)
+
+Let me refresh your memory: `apt` was a cli handling installation or removing the packages. It is a packaging tool. 'snap' - not the directory, the program - is similar, it is a packaging and deployment system. Snaps are a secure and scalable way to embed applications on Linux devices. A snap is an application **containerized** with all its dependencies. A snap can be installed using a single command on any device running Linux. Applications run fully isolated in their own sandbox, thus minimizing security risks. Thought, there was an incident, where in a snap, the developer put cryptocurrency mining software and all hell broke loose...
```

Comment:
Thought, there was an incident, where in a snap, the developer put cryptocurrency mining software and all hell broke loose...

did not understand the meaning of `Thought` in the beginning. Was it `think`, or was it `though`?

Reply:
though 🥲 Apparently, I don't know what I am writing while writing.